antiope-sqs/src/Antiope/SQS.hs

```diff
@@ -11,8 +11,10 @@ module Antiope.SQS
   , ackMessage
   , ackMessages
   , queueSource
+  , keepAliveMessage
   , forAllMessages
   , forAllMessages'
```

Comment:
nit: seems like hiding the double prime base function would aid clarity

Reply:
comment applied to the wrong line, I meant forAllMessages’’

Reply:
Yeah, I didn't need to expose it... Not for now at least... Had an idea of using it in the future, but not sure if it's going to be private or not..